haas/ext/obm/ipmi-obsolete.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2015-2016 Massachusetts Open Cloud Contributors
```

Comment:
Please replace the hyphen with an underscore. Sometimes python misconstrues the hyphen to be minus sign.
It will also create problems while importing.